decred-dcrdex/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+# DCRDEX uses its own auth
+      PROXY_AUTH_ADD: "false"  
+      APP_HOST: decred-dcrdex_web_1
+      APP_PORT: 5758
+  
+  web:
+    image: decred/dcrdex:v0.5.9
+    restart: on-failure
+    ports:
+      - 5758
+    volumes:
+      - ${APP_DATA_DIR}:/dex/.dexc
```

Comment:
Are you wanting to mount the entire app directory or do you just want a subdirectory to store data from `/dex/.dexc` in.

you could consider mounting just a data directory like this:
`- ${APP_DATA_DIR}/data:/dex/.dexc`